go.mod

```diff
@@ -454,3 +454,5 @@ replace (
 exclude k8s.io/client-go v12.0.0+incompatible
 
 exclude github.com/openshift/client-go v3.9.0+incompatible
+
+replace github.com/mitchellh/hashstructure => github.com/mitchellh/hashstructure/v2 v2.0.2
```

Comment:
Is there a reason you're not just importing this as `github.com/mitchellh/hashstructure/v2`?

Reply:
Not really. I'll change it to use `v2` directly.